.github/workflows/release.yml

```diff
@@ -0,0 +1,74 @@
+name: Release
+
+on:
```

Comment:
@guillaq those changes do not align with the MR title, were they introduced by accident ?

Reply:
other than that LGTM